products/BellHybrid/apps/common/include/common/widgets/ClockVertical.hpp

```diff
@@ -17,8 +17,8 @@ namespace style::time_set_spinner_vertical
         inline constexpr auto size = 6U;
     } // namespace focus
 
-    inline constexpr auto small_margin = 6U;
-    inline constexpr auto big_margin   = 44U;
+    inline constexpr auto small_margin    = 6U;
+    inline constexpr auto big_margin      = 44U;
     inline constexpr auto colossal_margin = -40;
```

Comment:
I think those might be no longer needed as they were used in spaceMarginsMap which is deleted

Reply:
Removed